src/tests/integration/omneo/regions/list-regions.test.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, expect, test, afterAll } from 'vitest'
+import { Omneo } from '../../../../omneo'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import { RegionInput, RegionResponse } from '../../../../types'
+import { getName, getHandle } from './util'
+
+const omneo = new Omneo({
+  tenant: process.env.OMNEO_TENANT as string,
+  token: process.env.OMNEO_TOKEN as string
+})
+const CREATED_REGIONS_IDS : number[] = []
+
+describe('Regions list', () => {
+  test('SDK Get Regions', async () => {
+    const payload: RegionInput = {
+      name: getName(),
+      handle: getHandle()
+    }
+    const response = await simpleOmneoRequest('POST', '/regions', payload)
+    CREATED_REGIONS_IDS.push(response.data.id)
+
+    const regionsRes: RegionResponse = await omneo.regions.list({
+      'filter[handle]': payload.handle
+    })
+    const { data: regions } = regionsRes
```

Comment:
expect regions.length to equal 1
It would be incorrect if more than one region was returned from your filter. 
since you've just created one with a random handle only 1 should ever exist

Reply:
fixed.